libweb3jsonrpc/Debug.cpp

```diff
@@ -134,3 +137,34 @@ Json::Value Debug::debug_storageAt(string const& _blockHashOrNumber, int _txInde
 		ret[toHex(toCompactBigEndian(i.first, 1), 2, HexPrefix::Add)] = toHex(toCompactBigEndian(i.second, 1), 2, HexPrefix::Add);
 	return ret;
 }
+
+Json::Value Debug::debug_traceCall(Json::Value const& _call, std::string const& _blockNumber, Json::Value const& _options)
+{
+	Json::Value ret;
+	try
+	{
+		Block temp = m_eth.asOf(jsToBlockNumber(_blockNumber));
+		TransactionSkeleton ts = toTransactionSkeleton(_call);
+		if (!ts.from) {
+			ts.from = m_ethAccounts.defaultTransactAccount();
```

Comment:
debug should not have access to the accounts, that is a security risk. If the from account is not set, just send from `u256(-1)`.